interface/forms/requisition/new.php

```diff
@@ -0,0 +1,363 @@
+             */
+               $lab_id = $order[0];
+               $storeBar = getBarId($lab_id,$pid);
+
+            if (!empty($storeBar)) {
+                $bar = $storeBar['req_id'];
+            } else {
+                $bar = rand(1000,999999);
+                saveBarCode($bar,$pid,$order[0]);
+            }
+
+            ?>
+            <img  src="../../forms/requisition/barcode.php?text=<?php echo attr($bar); ?>" alt="barcode" /></br>
+        &#160;&#160;&#160;&#160;&#160;	<?php echo text($bar); ?>
+        </div>
+        <div class="reqHeader" id="printableArea">
+        <p><font size="4"><b><?php print xlt('Requisition Number') ?>:</b> <?php echo text($bar); ?>  &#160;&#160;&#160;&#160;&#160;&#160;<b><?php print xlt('Client Number') ?>:</b> <?php echo text($provLabId['recv_fac_id']); ?></font></p>
+           <div class="cinfo">
+           <font size="4">
+               <?php echo text($facility['name']) ."<br>". text($facility['street']) . "<br>" .
+                          text($facility['city']).",".text($facility['state']).",".text($facility['postal_code']) . "<br>" .
+                          text($facility['phone']); ?>
+                          </font>
+           </div>
+           <div class="pdata">
+                 <p><font size="4">
+          <?php echo text($pp['organization'])."</br>".
+          text($pp['street'])." | ".text($pp['city']).", ".text($pp['state'])." ".text($pp['zip'])."</br>".
+          "O:".text($pp['phone'])." | F:".text($pp['fax'])."</br>";
+                 ?></font></p>
+
+           </div>
+        </div>
+        <div class="req" id="printableArea">
+            <table class="table" style="width:800px" border="1">
+               <tr style="height:125px;">
+                   <td style="vertical-align:top; width:400px;" >
+                   <div class="plist">
+                       <?php echo xlt('Collection Date/Time')?>:</br>
+                       <?php echo xlt('Lab Reference ID') ?>:</br>
+                       <?php echo xlt('Fasting')?>:</br>
+                       <?php echo xlt('Hours')?>:</br>
+                     </div>
+                    <div class="pFill">
+                      <?php echo text($order[6]);?> </br>
+                        <?php echo text($order[0]);?>
+                    </div>
+                   </td>
+                   <td style="vertical-align:top width: 800px">
+                    <div class="plist">
+                       <b><?php echo xlt('Patient ID') ?>: </b>  </br>
+                       <b><?php echo xlt('DOB') ?>: </b> </br>
+                       <b><?php echo xlt('Sex') ?>: </b>    </br>
+                       <b><?php echo xlt('Patient Name') ?>: </b>  </br>
+                    </div>
+                    <div class="pFill">
+                        <?php echo text($pid); ?></br>
+                        <?php echo text($pdata['DOB']); ?></br>
+                        <?php echo xlt($pdata['sex']); ?></br>
```

Comment:
This is a "list_options" item, so should do the following in order to present the list_title:
``````
<?php echo text(getListItemTitle('sex',$pdata['sex'])); ?></br>
``````

Reply:
(btw, above function is in the library/options.inc.php library)

Reply:
I have to push back on this one. This is selecting the sex that has been saved in the patient table data. This is not a drop-down list for the user to select from.

Reply:
hi @juggernautsei ,
When a list_options list item is save, the `option_id` is saved in the database. And then when we display it, we convert it to the `title`. The getListItemTitle function does this. This allows several things:
1. Makes translation an option (translations of lists can be turned off by users that decide to place their translations in the `title`.
2. Allows use of openemr in a multilingual clinic since the option_id is stored in database rather than the title.
Let me know if I need to provide more details on this.
thanks,
-brady